src/services/db/GitFileSystemService.ts

```diff
@@ -706,6 +793,49 @@ export default class GitFileSystemService {
     })
   }
 
+  mergeStagingToMaster(
+    repoName: string
+  ): ResultAsync<boolean, GitFileSystemError> {
+    const efsVolPath = this.getEfsVolPathFromBranch("staging")
+    return this.isValidGitRepo(repoName, "staging").andThen((isValid) => {
+      if (!isValid) {
+        return errAsync(
+          new GitFileSystemError(
+            `Folder "${repoName}" for EFS vol path: "${efsVolPath}" is not a valid Git repo`
+          )
+        )
+      }
+
+      return (
+        this.ensureCorrectBranch(repoName, "master")
+          .andThen(() =>
+            ResultAsync.fromPromise(
+              this.git
+                .cwd({ path: `${efsVolPath}/${repoName}`, root: false })
+                .merge(["staging", "--no-ff"]), // ensure there is a merge commit
```

Comment:
no longer using git merge

Reply:
my bad, I meant that this method isn't being used anymore as I am pulling from origin/master instead of merging in staging locally

just to avoid confusion, I've removed this function from the PR
[27d6435](https://github.com/isomerpages/isomercms-backend/pull/1172/commits/27d643548cec3739e018faa88a04e97a76e6e23a)